package.json

```diff
@@ -73,7 +73,7 @@
   "dependencies": {
     "@fortawesome/fontawesome-svg-core": "^1.2.25",
     "@fortawesome/free-regular-svg-icons": "^5.11.2",
-    "@fortawesome/free-solid-svg-icons": "^5.11.2",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
```

Comment:
Out of curiosity, was this bump needed for the spinner?

Reply:
It looks like it wasn't needed. I reverted it back and the spinner still shows.